src/tests/lib/components/DropdownTest.svelte

```diff
@@ -10,6 +10,7 @@
   };
 </script>
 
+<!-- eslint-disable svelte/valid-compile -->
```

Comment:
What's this?

Reply:
Disabling an eslint rule in a test component for a test